### ISSUE TITLE
Fail with filename contain spaces

### ISSUE THREAD

**Issue by MicroBeast** [kanryu/quickviewer#46]
Honestly I'm just testing your application as I found it by accident on opendesktop.org, so no further investigation.
Aside, I'm not a programmer.

QuickView 1.0.0 x86_64 AppImage.
Linux/Debian sid amd64

Trying to open a comic archive with name contain spaces fail.
**Application error message:** Image file not found. Can't be open.
**Console error message:** Qt7z: File does not exist: "space.[cbz|zip]". That if for example the name was "file name with space.[cbz|zip]". Only the last part taken. No matter if it archive or plain image.

This error cause this application can't be used as open with (xdg mime), from script or terminal.
Opening the same file from the application "File>Open Image Folder" works however.

---
Side note:
Trying to build the entire thing my self fail around unrar stuff. Maybe because I pass those said needed AppImage libraries. I don't use any QT IDE or GUI build helper like mentioned in your how to build notes.
No rar/unrar libraries found in Debian repo except *rar/unrar* proprietary application.

Thank you.

**Comment by kanryu** [comment]
QuickViewer has two ways to open the archive. It is a method of dragging and dropping from the file manager onto the window and a method of selecting [File] -> [Open Image Folder].

I have been constantly testing with kubuntu - 14.04.5 and ubuntu - 14.04.5 amd64, but archives containing spaces (including cbr / cbz) work fine.

Therefore, there may be an error in the way you specify the archive file in QuickViewer.
For example, file names containing spaces on the console will have to be pinched by "double quotes" or "single quotes".

Build procedure:

Everything is written in HowToBuild.md.
Basically, it assumes that you download and install Qt SDK. Qt IDE is unnecessary.

Qt environment by a general Unix package manager (eg apt) seems to be remarkably old version. It is still almost Qt 4's place. Qt 5 is necessary to build QuickViwer. 

Do not forget to run 'git submodule init / update' before building.

**Comment by MicroBeast** [comment]
I also already tested with double/single quotes. Doesn't work either, same error.

I need hours to figure out what's QT SDK if seen from Debian perspective. Blind for anything QT here.
I'll recheck my QT dev environment. Debian sid/unstable does already have an army of QT5 dev packages.

Good job for the application anyway. The fastest for it's kind in Linux world as far I could find.

**Comment by kanryu** [comment]
@MicroBeast  Thank you.
I had not tested that AppImage passed the file path from the command line.

I have released rev2 so please download it again :)

**Comment by MicroBeast** [comment]
1. Excellent for the quick fix. Only few hours? I don't even yet re-study your codes. Real life win.
2. Second excellent because it does fix the issue.
  Using from console/terminal:
 - **QuickView "file name.cbz"** work. 
 - **QuickView file\ name.cbz** (unix traditional space escaping) work.
 - Naturally, also work on linux xdg/mimetype (used by file manager) system.
